optaplanner-examples/src/test/resources/org/optaplanner/examples/tsp/data/dj38.xml

```diff
@@ -0,0 +1,352 @@
+            <location class="TspAirLocation" reference="18"/>
+        </Visit>
+        <Visit id="58">
+            <id>17</id>
+            <location class="TspAirLocation" reference="19"/>
+        </Visit>
+        <Visit id="59">
+            <id>18</id>
+            <location class="TspAirLocation" reference="20"/>
+        </Visit>
+        <Visit id="60">
+            <id>19</id>
+            <location class="TspAirLocation" reference="21"/>
+        </Visit>
+        <Visit id="61">
+            <id>20</id>
+            <location class="TspAirLocation" reference="22"/>
+        </Visit>
+        <Visit id="62">
+            <id>21</id>
+            <location class="TspAirLocation" reference="23"/>
+        </Visit>
+        <Visit id="63">
+            <id>22</id>
+            <location class="TspAirLocation" reference="24"/>
+        </Visit>
+        <Visit id="64">
+            <id>23</id>
+            <location class="TspAirLocation" reference="25"/>
+        </Visit>
+        <Visit id="65">
+            <id>24</id>
+            <location class="TspAirLocation" reference="26"/>
+        </Visit>
+        <Visit id="66">
+            <id>25</id>
+            <location class="TspAirLocation" reference="27"/>
+        </Visit>
+        <Visit id="67">
+            <id>26</id>
+            <location class="TspAirLocation" reference="28"/>
+        </Visit>
+        <Visit id="68">
+            <id>27</id>
+            <location class="TspAirLocation" reference="29"/>
+        </Visit>
+        <Visit id="69">
+            <id>28</id>
+            <location class="TspAirLocation" reference="30"/>
+        </Visit>
+        <Visit id="70">
+            <id>29</id>
+            <location class="TspAirLocation" reference="31"/>
+        </Visit>
+        <Visit id="71">
+            <id>30</id>
+            <location class="TspAirLocation" reference="32"/>
+        </Visit>
+        <Visit id="72">
+            <id>31</id>
+            <location class="TspAirLocation" reference="33"/>
+        </Visit>
+        <Visit id="73">
+            <id>32</id>
+            <location class="TspAirLocation" reference="34"/>
+        </Visit>
+        <Visit id="74">
+            <id>33</id>
+            <location class="TspAirLocation" reference="35"/>
+        </Visit>
+        <Visit id="75">
+            <id>34</id>
+            <location class="TspAirLocation" reference="36"/>
+        </Visit>
+        <Visit id="76">
+            <id>35</id>
+            <location class="TspAirLocation" reference="37"/>
+        </Visit>
+        <Visit id="77">
+            <id>36</id>
+            <location class="TspAirLocation" reference="38"/>
+        </Visit>
+        <Visit id="78">
+            <id>37</id>
+            <location class="TspAirLocation" reference="39"/>
+        </Visit>
+        <Visit id="79">
+            <id>38</id>
+            <location class="TspAirLocation" reference="40"/>
+        </Visit>
+    </visitList>
+</TspSolution>
```

Comment:
There seems to be an identical file in `optaplanner-examples/data/tsp/unsolved`. Can we reuse it?